internal/configs/import.go

```diff
@@ -0,0 +1,13 @@
+package configs
+
+import (
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/terraform/internal/addrs"
+)
+
+type Import struct {
+	ID string
+	To addrs.AbsResourceInstance
```

Comment:
I think this is OK for now since it's static, but we may need to have something similar to `addrs.MoveEndpoint`, since we'll probably be doing the same kind of addres-overloading.

Reply:
Yes, I originally had this as `addrs.MoveEndpoint`, but I think it's better to keep it as the basic static addr for now and create a custom type for importable addrs later.